src/dependency-manager/spec/utils/slugs.ts

```diff
@@ -20,6 +20,8 @@ export class Slugs {
   public static INSTANCE_DELIMITER = '@';
   public static SLUG_CHAR_LIMIT = 32;
 
+  public static DB_SUFFIX = '-managed-db';
```

Comment:
I'm not sold on this name since we don't manage the db in the preview case. Would prefer `-db`, but I still don't love that someone could create a conflict via:

```yaml
databases:
  mysql:

services:
  mysql-db:
```

Obviously unlikely that someone names the services in this way. If we are going to move forward with this services block merging then we should at least error if a conflict is generated.